bellatrix.core/src/main/java/solutions/bellatrix/core/utilities/ConverterService.java

```diff
@@ -0,0 +1,58 @@
+package solutions.bellatrix.core.utilities;
+
+import lombok.SneakyThrows;
+
+public class ConverterService {
+    // ToDo Dozer must be used instead
+    @SneakyThrows
+    public static <SourceT, ResultT> ResultT convert(SourceT source, int index) {
+        ResultT object = InstanceFactory.createByTypeParameter(source.getClass(), index);
+
+        return convert(source, object);
+    }
+
+    // ToDo Dozer must be used instead
+    @SneakyThrows
```

Comment:
I would suggest not to use Sneaky Throws unless its really needed - It is more useful to add try catch and provide meaningful message than swallowing the error